_data/sites.json

```
@@ -7353,6 +7353,16 @@
         ]
     },
 
+    {
+        "name": "GermanTechJobs",
+        "url": "https://germantechjobs.de/contact",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "hard",
```
Even if the deletion is quick, contacting support is considered `hard` by us due to having to wait someone to actually do it.